src/services/utils.ts

```diff
@@ -10,4 +12,15 @@ function formatDateTime(date: Date) {
   return formatter.format(date);
 }
 
-export { formatDateTime };
+async function parseJsonFromBody(request: Request): Promise<any> {
+  let body = undefined;
+  try {
+    body = await request.json();
+    return body;
+  } catch (err) {
+    console.error(err);
+    return response400("Body needs to be in JSON format");
```

Comment:
Did you test this function? Meaning, the situation in which the body is not JSON?